### ISSUE TITLE
why mAP of my train model is under mAP of this paper？

### ISSUE THREAD

**Issue by MillX2021** [HikariTJU/LD#12]
I download Resnet101 GFocal model from mmdetction（download link：https://download.openmmlab.com/mmdetection/v2.0/gfl/gfl_r101_fpn_mstrain_2x_coco/gfl_r101_fpn_mstrain_2x_coco_20200629_200126-dd12f847.pth), and train by this command line：
bash ./tools/dist_train.sh configs/ld/ld_gflv1_r101_r50_fpn_coco_1x.py 8
but I get the mAP of epoch 12 is 0.3960。this paper give this mAP should be 41.2。
train and eval log：
2021-08-18 10:02:25,727 - mmdet - INFO - Epoch [12][4750/4887]	lr: 3.750e-05, eta: 0:02:01, time: 0.882, data_time: 0.015, memory: 6289, loss_cls: 0.3524, loss_bbox: 0.2883, loss_ld: 0.1610, loss_dfl: 0.2044, loss: 1.0061
2021-08-18 10:03:10,220 - mmdet - INFO - Epoch [12][4800/4887]	lr: 3.750e-05, eta: 0:01:17, time: 0.890, data_time: 0.014, memory: 6289, loss_cls: 0.3597, loss_bbox: 0.2918, loss_ld: 0.1602, loss_dfl: 0.2045, loss: 1.0162
2021-08-18 10:03:54,650 - mmdet - INFO - Epoch [12][4850/4887]	lr: 3.750e-05, eta: 0:00:32, time: 0.888, data_time: 0.015, memory: 6289, loss_cls: 0.3543, loss_bbox: 0.2908, loss_ld: 0.1611, loss_dfl: 0.2060, loss: 1.0122
2021-08-18 10:04:45,038 - mmdet - INFO - Saving checkpoint at 12 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 5000/5000, 95.6 task/s, elapsed: 52s, ETA:     0s

2021-08-18 10:05:50,605 - mmdet - INFO - Evaluating bbox...
Loading and preparing results...
DONE (t=6.00s)
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=53.03s).
Accumulating evaluation results...
DONE (t=11.64s).
Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.396
Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=1000 ] = 0.572
Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=1000 ] = 0.428
Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.226
Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = 0.435
Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = 0.516
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.582
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=300 ] = 0.582
Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=1000 ] = 0.582
Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=1000 ] = 0.369
Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=1000 ] = 0.630
Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=1000 ] = 0.745
2021-08-18 10:07:02,558 - mmdet - INFO - Exp name: ld_gflv1_r101_r50_fpn_coco_1x.py
2021-08-18 10:07:02,559 - mmdet - INFO - Epoch(val) [12][4887]	bbox_mAP: 0.3960, bbox_mAP_50: 0.5720, bbox_mAP_75: 0.4280, bbox_mAP_s: 0.2260, bbox_mAP_m: 0.4350, bbox_mAP_l: 0.5160, bbox_mAP_copypaste: 0.396 0.572 0.428 0.226 0.435 0.516

could you help me to slove this problem？

**Comment by HikariTJU** [comment]
You should edit config file according your gpu num and samples_per_gpu, please refer to "Learning rate setting" part in readme.

For 8 gpus, I recommend using 
samples_per_gpu=2
lr=0.01

**Comment by MillX2021** [comment]
thanks. I'm going to recurrence it.

**Comment by Zzh-tju** [comment]
> I download Resnet101 GFocal model from mmdetction（download link：https://download.openmmlab.com/mmdetection/v2.0/gfl/gfl_r50_fpn_mstrain_2x_coco/gfl_r50_fpn_mstrain_2x_coco_20200629_213802-37bb1edc.pth)

Are you sure you download R101 GFocal model？

**Comment by MillX2021** [comment]
> > I download Resnet101 GFocal model from mmdetction（download link：https://download.openmmlab.com/mmdetection/v2.0/gfl/gfl_r50_fpn_mstrain_2x_coco/gfl_r50_fpn_mstrain_2x_coco_20200629_213802-37bb1edc.pth)
> 
> Are you sure you download R101 GFocal model？

hhh, i quote error.but I download r1010 gfocal model from https://download.openmmlab.com/mmdetection/v2.0/gfl/gfl_r101_fpn_mstrain_2x_coco/gfl_r101_fpn_mstrain_2x_coco_20200629_200126-dd12f847.pth.
but I get the mAP is 40.8(the mAP is 41.2 in this paper).

**Comment by Zzh-tju** [comment]
@MillX2021 
41.2 AP in the paper is evaluated on coco test-dev 2019.
For val2017, AP is 41.1
